server/src/models/results/populate.js

```diff
@@ -48,10 +50,10 @@ export default async function({models}){
       doc,
       stable_id,
     } = indicator;
-    
+
     indicator.indicator_id = indicator.id;
     indicator.id = null;
-    indicator.target_year = _.isNaN(parseInt(target_year)) ? null : parseInt(target_year);
+    indicator.target_year = null_preserving_to_number( _.parseInt(target_year) );
     indicator.target_month = _.isEmpty(target_month) ? null : +target_month;
```

Comment:
`_.isEmpty` and `null_preserving_to_number` is different (I think mainly for `0` values) so I left it as is

Reply:
I think they give the same value? What were you seeing that was different?

At this point, `target_month` should be a string or null, so `_.isEmpty` will only be true if the value is null or an empty string `""`.

Reply:
So as of current `null_preserving_to_number`, it will output `0`,  where as isEmpty gives `null`. which now that I look back, is actually error from `null_preserving_to_number` as it should be `null`. I'll just be removing this function so there's no confusion..!